index.js

```diff
@@ -71,7 +73,6 @@ const createAdapter = handler => config => new Promise((resolve, reject) => {
     }
   ).then(
     (response) => {
-      server.off('error', reject)
```

Comment:
Why does it removed?

Reply:
Because this code can fix the server's repeated listening problem.
```js
if (server.eventNames().includes('error')) {
    server.on('error', reject)
  }
```

Reply:
Can you add a teat case to describe your issue?

Reply:
That is my error, I mean code is this

```js
if (!server.eventNames().includes('error')) {
    server.on('error', reject)
}
```
I will write test case to describe my issue.

Reply:
my test case: https://github.com/pagewang0/axiosist/blob/master/test.js#L197 @Gerhut